build.py

```diff
@@ -124,7 +125,7 @@ def Run(context, cmd):
     """Run the specified command in a subprocess."""
     PrintInfo('Running "{cmd}"'.format(cmd=cmd))
 
-    with open(context.logFileLocation, "a") as logfile:
+    with codecs.open(context.logFileLocation, "a", "utf-8") as logfile:
```

Comment:
I had a fix locally for it as well...but I didn't have to change anything else than line 150, i.e. remove encode and write simply content of `l`. 

How can I see the issue which forced you to use codecs?

Reply:
`I had a fix locally for it as well...but I didn't have to change anything else than line 150, i.e. remove encode and write simply content of l.`

If I recall correctly, I had the code originally that way and it was working on some platforms but not others.

`How can I see the issue which forced you to use codecs?`
Use python 3 to run the build script. 

e.g
`C:\Python37\python.exe build.py .....`

Reply:
That is exactly what I'm doing...and all I needed is a change to switch from `logfile.write(str(l.encode("utf8")))` to `logfile.write(l)`.

Reply:
Yeah, I see but we had random failures with that approach.  I think the codecs approach should be fine. I did a few tests and seems to be working.

Please give it a try and see if it solves things for you as well.

Reply:
From offline discussion - this looks to be platform-specific and this is why I only required a smaller fix on MacOS.